Add `Tournament Committee applications now open!` news post (#12659)
ppy/osu-wiki: news/2025/2025-01-13-tournament-committee-applications-now-open.md

---
layout: post
title: Tournament Committee Applications Now Open!
date: 2025-01-13 15:00:00 +0000
---

Calling all tournament goers! We're excited to announce that, for the first time ever, formal applications for the [Tournament Committee (TC)](/wiki/People/Tournament_Committee) are now open!

![](/wiki/shared/news/2025-01-13-tournament-committee-applications-now-open/banner.jpg)

We are seeking passionate individuals with significant tournament experience — including players — to help support and improve the community.

## Who are we, and what do we do?

The Tournament Committee is a panel of skilled and established organisers and helpers in the tournament scene. Here is some of what we do:

- Reviewing tournaments for badge support.
- Handling tournament reports and inquiries from the community.
- Keeping rules up to date in the [tournament support](/wiki/Tournaments/Official_support) wiki page.
- Mentoring event organisers, answering questions, and providing resources.

## Requirements

To join the TC, you must meet the following minimum criteria:

> - **Age:** You must be at least 18 years of age.
> - **Account standing:** All applicants are checked for any notable community rule violations within the last 12 months, which would disqualify them from joining.
> - **Language proficiency:** Fluency in English is mandatory, as most reviewed content and all communications are in English.

Reviewer: maybe remove these now to be consistent

```suggestion
- You must be at least 18 years of age.
- All applicants are checked for any notable community rule violations within the last 12 months, which would disqualify them from joining.
- Fluency in English is mandatory, as most reviewed content and all communications are in English.
```

---
layout: post
title: Tournament Committee Applications Now Open!
date: 2025-01-13 15:00:00 +0000
---

Calling all tournament goers! We're excited to announce that, for the first time ever, formal applications for the [Tournament Committee (TC)](/wiki/People/Tournament_Committee) are now open!

![](/wiki/shared/news/2025-01-13-tournament-committee-applications-now-open/banner.jpg)

We are seeking passionate individuals with significant tournament experience — including players — to help support and improve the community.

## Who are we, and what do we do?

The Tournament Committee is a panel of skilled and established organisers and helpers in the tournament scene. Here is some of what we do:

> - **Tournament reviews:** Approving or denying badge support for tournaments and communicating any necessary changes to organisers.
> - **Handling reports and inquiries:** Investigating tournament-related issues reported by the community, assessing the validity of claims, and providing timely and well-considered responses to users. 
> - **Maintaining the [tournament support](/wiki/Tournaments/Official_support) wiki page:** Keeping official rules and documentation up to date.
> - **Mentoring the community:** Guiding newer members of the tournament scene towards making better decisions.
> - **Community leadership:** Acting as a representative and leader within the tournament space.

Reviewer: this list format is very formal. you could do away with the bolded titles and the verbosity:

```suggestion
- Reviewing tournaments for badge support.
- Handling tournament reports and inquiries from the community.
- Keeping rules up to date in the [tournament support](/wiki/Tournaments/Official_support) wiki page.
- Mentoring event organisers, answering questions and providing resources.
```

however, i don't particularly mind this one as it is

Author: I can see how it's formal especially compared to the rest of the lists. I think oxford comma is needed on that last line  as we use it elsewhere in the post.

Reviewer: edited, didn't notice that woops

> ## Why join us?
> 
> - **Make a Difference:** Your efforts will directly impact the quality and success of the osu! tournament community.
> - **Develop Skills:** Gain experience in management, teamwork, and leadership.
> - **Connect with the Community:** Work alongside other passionate community members and make lasting friendships and connections.
> 

Reviewer: the reason this article reads as corporate is not just a bit of stiff wording. it's mainly the usage of verbose lists

this one is just fluff and comes across as pompous

```suggestion
## Requirements

```

To join the TC, you must meet the following minimum criteria:

- **Age:** You must be at least 18 years of age.

> - **Account standing:** All applicants will undergo a security check. Any notable disciplinary actions within the last 12 months will disqualify a candidate.

Reviewer: ```suggestion
- **Account standing:** All applicants are checked for any notable community rule violations within the last 12 months, which would disqualify them from joining.
```

- **Language proficiency:** Fluency in English is mandatory, as most reviewed content and all communications are in English.

## Who should apply?

We're looking for individuals who are:

- Passionate about osu! tournaments
- Team players
- Exceptional communicators

> - Organized and reliable

Reviewer: ```suggestion
- Organised and reliable
```
